### DISCUSSION TITLE
Cant able to enable the ingress in stackstom which is deployed in HA mode

### DISCUSSION THREAD

**Discussion by itsmano1993** [StackStorm/st2#5290] [Operations]
### STACKSTORM VERSION

3.4-dev:

##### OS, environment, install method

aws eks cluster

![Screenshot from 2021-06-15 14-54-00](https://user-images.githubusercontent.com/17376605/122373455-20061d80-cf7f-11eb-86d6-10e6ecb8997f.png)

Thanks!

**Comment by itsmano1993** [comment]
Hi Team, Any thoughts on my above query?

**Reply by arms11** [reply]
Asking this to understand it bit better. 
1. Since this is in AWS, are you using `Ingress` resource with preferably `ALB`?
2. If you are declaring `st2web` to be of `LoadBalancer` type instead of `NodePort`, AWS should have already provisioned `CLB`. Please check the `st2web` in services. If you find the ExternalIP there, that is what you can use as `Host` here.

**Reply by itsmano1993** [reply]
Hi @arms11 Thanks for your ask, 
I did not set anything specific to the load balancer side. This is my requirement,
Our application server will invoke the stackstom APIs, auth module. 
By default when we deployed the stackstorm in HA mode we are getting the IP like this http://54.xxx.xxx.xxx:30042/
I can use this IP and it's doing the reverse proxy perfectly. but this port will get changed when we deployed again.

1. instead of port 30042 if I want to use 80 what is the change should we make?
2. I am seeing reverse proxy is working fine, by default st2 web create load balancer also?
I am using this helm chart for the deployment - [https://github.com/StackStorm/stackstorm-ha/blob/master/values.yaml]

Thanks
Mano

**Reply by itsmano1993** [reply]
Missed answering your question
My current deployed st2web service its type is Node port only
